packages/fast-components-styles-msft/src/breadcrumb-item/index.ts

```diff
@@ -0,0 +1,49 @@
+import designSystemDefaults, {
+    DesignSystem,
+    withDesignSystemDefaults,
+} from "../design-system";
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import { ensureBrandNormal, ensureForegroundNormal } from "../utilities/colors";
+import { BreadcrumbItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { applyTypeRampConfig } from "../utilities/typography";
+import { fontWeight } from "../utilities/fonts";
+import { toPx } from "@microsoft/fast-jss-utilities";
+
+function applyHypertextBorder(pixels: number): CSSRules<DesignSystem> {

```

Comment:
We shouldn't need these styles.

Reply:
@eljefe223 I don't know that we need these - but even if we did, I would expect us to import the hypertext styles for instead of duplicating.